sage-ios/SAGE/SAGE/ProfileView.swift

```diff
@@ -156,11 +224,27 @@ class ProfileView: UIView {
         self.header.setHeight(self.headerHeight+self.headerOffset)
         
         // position edit button
-        self.profileEditButton.text = "Edit Profile"
+        self.profileEditButton.layoutIfNeeded()
         self.profileEditButton.sizeToFit()
         self.profileEditButton.setY(self.headerHeight + 15)
         let profileEditButtonX = CGRectGetWidth(self.header.frame) - self.leftMargin - CGRectGetWidth(self.profileEditButton.frame)
         self.profileEditButton.setX(profileEditButtonX)
+        
+        // position promote button
```

Comment:
just set the promote and demote button to be the same widths. it looks better because when you press either button itll look like the same button because the width isnt jumping